strr-strata-web/app/pages/application.vue

```diff
@@ -0,0 +1,139 @@
+<script setup lang="ts">
+import { ConnectStepper, FormPlatformReviewConfirm } from '#components'
+
+const { t } = useI18n()
+
+const { validateContact } = useStrrContactStore()
+const { validateStrataBusiness } = useStrrStrataBusinessStore()
+const { validateStrataDetails } = useStrrStrataDetailsStore()
+const { validateStrataConfirmation } = useStrrStrataApplicationStore()
+// fee stuff
+const {
+  addReplaceFee,
+  getFee,
+  setPlaceholderFilingTypeCode
+} = useConnectFeeStore()
+
+const strataFee = ref<ConnectFeeItem | undefined>(undefined)
+
+onMounted(async () => {
+  // TODO: check for application id in the route query, if there then load the application
+  strataFee.value = await getFee(StrrFeeEntityType.STRR, StrrFeeCode.STR_STRATA)
+  if (strataFee.value) {
+    addReplaceFee(strataFee.value)
+  } else {
+    // error getting fee info from api
+    // TODO: set fee to a static value or set an error in the fee summary?
+    setPlaceholderFilingTypeCode(StrrFeeCode.STR_STRATA)
+  }
+})
+
+// stepper stuff
+// TODO: replace validation functions
+const steps = ref<Step[]>([
+  {
+    i18nPrefix: 'strr.step',
+    icon: 'i-mdi-domain-plus',
+    complete: false,
+    isValid: false,
+    validationFn: () => validateStrataBusiness(true) as boolean
+  },
+  {
+    i18nPrefix: 'strr.step',
+    icon: 'i-mdi-account-multiple-plus',
+    complete: false,
+    isValid: false,
+    validationFn: async () => await validateContact(true) as boolean
+  },
+  {
+    i18nPrefix: 'strr.step',
+    icon: 'i-mdi-map-marker-plus-outline',
+    complete: false,
+    isValid: false,
+    validationFn: () => validateStrataDetails(true) as boolean
+  },
+  {
+    i18nPrefix: 'strr.step',
+    icon: 'i-mdi-text-box-check-outline',
+    complete: false,
+    isValid: false,
+    validationFn: () => validateStrataConfirmation(true) as boolean
+  }
+])
+const activeStepIndex = ref<number>(0)
+const activeStep = ref<Step>(steps.value[activeStepIndex.value] as Step)
+const stepperRef = shallowRef<InstanceType<typeof ConnectStepper> | null>(null)
+const reviewFormRef = shallowRef<InstanceType<typeof FormPlatformReviewConfirm> | null>(null)
+
+// TODO: musing - should we move this into the stepper component and add button items to the 'Step' object
```

Comment:
This is something to consider, I guess the question is how tightly coupled do we want these 2 components (stepper/button control)?